contracts/Comptroller.sol

```diff
@@ -947,6 +951,32 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
         allMarkets.push(CToken(cToken));
     }
 
+    function _initializeMarket(address cToken) internal {
+        uint32 blockNumber = safe32(getBlockNumber(), "block number exceeds 32 bits");
+
+        CompMarketState storage supplyState = compSupplyState[cToken];
+        CompMarketState storage borrowState = compBorrowState[cToken];
+
+        /*
+         * Update market state indices
+         */
+
+        if (supplyState.index == 0) {
```

Comment:
If this is only called for new markets, why would this not be 0?

Reply:
If a market is added, then removed, then added again, we don't want to re-initialize the supply index.

It looks like it's not currently possible to remove a market, but just in-case it is in the future, I put that check there.

Reply:
Hm ok guess that makes sense. Do we have either an extra space after the comment above or not enough on the one below? 😝

Reply:
It's free real estate